lib/expeditor/command.rb

```diff
@@ -43,7 +43,7 @@ def started?
     def get
       raise NotStartedError if not started?
       @normal_future.get_or_else do
-        if @fallback_var
+        if @fallback_var && @service.fallback_enabled?
```

Comment:
In this file, `@fallback_var` is referenced in other places. Are you sure `@service.fallback_enabled?` should affect only this `@fallback_var`?